src/mscorlib/src/System/Text/Encoding.cs

```diff
@@ -884,6 +884,19 @@ public virtual int GetByteCount(String s)
         [Pure]
         public abstract int GetByteCount(char[] chars, int index, int count);
 
+        // Returns the number of bytes required to encode a string range.
+        //
+        [Pure]
+        public virtual int GetByteCount(string s, int index, int count)
```

Comment:
please remove the virtual and if anyone want it back (which I don't think anyone will do) we can still get it back. 

---
In reply to: [92673324](https://github.com/dotnet/coreclr/pull/8651#discussion_r92673324) [](ancestors = 92673324)